packages/mobile/locales/en-US/global.json

```diff
@@ -113,5 +113,6 @@
   "timeframes": {
     "30d": "30 days"
   },
-  "message": "Message"
+  "message": "Message",
+  "paymentLimitReached": "The daily limit is {{dailyLimit}} {{dailyRemaining}} remaining"
```

Comment:
I am not sure about how to pass these parameters but @aaronmgdr is looking into it.